tests/checkpoint/test_checkpoint_result_format.py

```diff
@@ -1,3445 +0,0 @@
-from __future__ import annotations
-
-import logging
```

Comment:
REWRITE - these tests are around ID/PK and should be rewritten if we want to keep this feature for V1